### PR TITLE
feat(timeline): allow logType for errors and warnings

### PR DESCRIPTION
**Pull request by posva**
Screen Shot 2020-09-02 at 15 22 53

### CODE REVIEW

**Comment by Akryum** [comment]
Did you try with the icons?

**Comment by posva** [comment]
Yeah, the circle is too small for the icon. Maybe a different one would work. We could also draw something different from a circle and differentiate by shape on top of color:

**Comment by Akryum** [comment]
Sounds like a good first implementation. Maybe you can delete the icon images then.

**Comment by posva** [comment]
Should be good as long as the PR is squashed

File: packages/app-frontend/src/features/timeline/TimelineView.vue [hunk 1]
```diff
@@ -125,11 +127,13 @@ export default {
       updateEventPosition(event, g)
       g.y = 16
       event.g = g
-      if (selectedEvent.value === event) {
-        drawSelectedEvent(event)
-      } else {
-        drawUnselectedEvent(event)
-      }
+      setTimeout(() => {
```
Comment (Akryum) [inline packages/app-frontend/src/features/timeline/TimelineView.vue:130]: Why the timeout?

Comment (posva) [reply packages/app-frontend/src/features/timeline/TimelineView.vue:130]: ah yeah, I forgot about this one and didn't put a comment! I realized that stackedEvents do not appear immediately, so some stacked events would render the first event meaning that one normal and an error would render as normal. I imagine the problem is somewhere else, where the call to `addEvent` is done but I didn't dig more 😅

**Pull request merged**